CHANGELOG.md

```diff
@@ -22,6 +24,10 @@ Contributors:
   ANSI dialect which defaults to the semicolon, but allows it to be more intuitive 
   when overriden in a specific child dialect (mysql) [#901](https://github.com/sqlfluff/sqlfluff/issues/901))
 - Added support for the DELIMITER statement in the mysql dialect [#901](https://github.com/sqlfluff/sqlfluff/issues/901))
+
+Contributors:
```

Comment:
I think you might have added this by mistake, (I'm seeing many issues managing CHANGELOGs on PRs :/ )

Reply:
Probably so. This file has conflicts all the time, because people are changing same/adjacent lines. I wonder if there's a way to avoid this somehow.

Reply:
Will remove.